src/main/java/core/basesyntax/figures/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.figures;
+
+public class IsoscelesTrapezoid extends Figure {
+
+    private final double firstBase;
+    private final double secondBase;
+    private final double height;
+
+    public IsoscelesTrapezoid(String color, double firstBase, double secondBase, double height) {
+        super(color);
+        this.firstBase = firstBase;
+        this.secondBase = secondBase;
+        this.height = height;
```

Comment:
According to the checklist, creating a figure, don't pass expressions in the constructor. Create separate variables and pass them on for better code readability. This applies to the constructor parameters `firstBase`, `secondBase`, and `height`. Ensure they are assigned from variables rather than directly from the method call arguments.